spec/lib/ak_action_creator_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+describe AkActionCreator do
+  let(:hostname) { 'http://localhost' }
+  let(:username) { 'fake_username' }
+  let(:password) { 'fake_password' }
+  let(:creator) { AkActionCreator.new hostname, username, password }
+
+  it 'should call the endpoint for creating an action in ActionKit' do
```

Comment:
According to Omar, you shouldn't use 'should' in it blocks :D Because they convey no information